infrasim/model.py

```diff
@@ -36,7 +36,7 @@
 import stat
 from . import logger, run_command, CommandRunFailed, ArgsNotCorrect, CommandNotFound, has_option
 
-TEMPLATE_ROOT = "/usr/local/etc/infrasim"
+TEMPLATE_ROOT = os.environ['HOME'] + "/.infrasim/"
```

Comment:
If we treat all files in infrasim folder as key libraries and someone else may consider incorprating infrasim from here, it would be more appropriate to pass variable TEMPLATE_ROOT from module that is calling into us.